airflow/www/security.py

```diff
@@ -44,6 +46,24 @@
 }
 
 
+class DefaultSessionInterface(SecureCookieSessionInterface):
+    """
+    Default cookie session interface.
+    This prevents creating flask sessions on REST API requests.
+    """
+
+    def save_session(self, *args, **kwargs):
+        """Prevent creating session from REST API requests."""
+        if g.get('login_from_api'):
+            return None
+        return super().save_session(*args, **kwargs)
+
+    @user_loaded_from_header.connect
+    def user_loaded_from_header(self, user=None):  # pylint: disable=unused-argument
+        """Set login_from_api in g"""
```

Comment:
This comment does't add any "value" -- we can see that it's doing this.

Either don't have a docstring here, or say _why_ we are setting this.

Additionlay: why do we need this and the change in airflow/api_connexion/security.py?

Reply:
This is actually for a case where login is configured to be processed through the header. Airflow did not configure this, but users may configure it. We can remove it. Should I?